### PR TITLE
feat: navigate to submissions page when clicked on the file icon in qr popup

### LINKED ISSUE #1243: Issues on QR pop up
**Describe the bug**

- [x] The download icon and view submission icon should be removed from start mapping QR pop-up.
image

- [ ] The download icon and view submission icon should have tool tips as in design.

- [x] The download QR icon should be functional. 
image

- [ ] The respective icons should lead to respective pop-up or page. 

The download icon should be linked with download mb tiles pop- up 
the view task submission icon should redirect to task submission page.

**To Reproduce**

1. Go to 'project page
2. Click on 'select Tasks to start mapping
3. See error

### PR DESCRIPTION
**Pull request by Prajwalism**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Issue

Fixes #1243 

## Describe this PR

This PR enables us to navigate to the submissions page when clicked on the `QR-popup` file icon with the `taskId` filter on.

## Screenshots

Please provide screenshots of the change.

## Checklist before requesting a review

- 📖 Read the FMTM Contributing Guide: <https://github.com/hotosm/fmtm/blob/main/CONTRIBUTING.md>
- 📖 Read the FMTM Code of Conduct: <https://github.com/hotosm/fmtm/blob/main/CODE_OF_CONDUCT.md>
- 👷‍♀️ Create small PRs. In most cases, this will be possible.
- ✅ Provide tests for your changes.
- 📝 Use descriptive commit messages.
- 📗 Update any related documentation and include any relevant screenshots.

### CODE REVIEW

**Review by varun2948 — APPROVED** [review]

File: src/frontend/src/components/ProjectDetailsV2/TaskSectionPopup.tsx [hunk 1]
```diff
@@ -74,7 +78,9 @@ const TaskSectionPopup = ({ taskId, body, feature }: TaskSectionPopupPropType) =
         <AssetModules.DescriptionOutlinedIcon
           style={{ width: '20px' }}
           className="hover:fmtm-text-primaryRed"
-          onClick={() => {}}
+          onClick={() => {
+            navigate(`/project-submissions/MTAwMTExMTE=?tab=table&task_id=${taskId}`);
```
Comment (spwoodcock) [inline src/frontend/src/components/ProjectDetailsV2/TaskSectionPopup.tsx:82]: Is this navigating to a hardcoded project url?
Shouldn't it be dynamic based on the current project page, or am I missing something?

**Comment by spwoodcock** [comment]
Looks like a merge got bodged somewhere 😄 
I would either:
- Copy your changes out somewhere, reset this branch, then copy them back in and force push.
- Rebase this branch with `development`:

```bash
git checkout development
git pull
git checkout feat/qr-popup-icons
git rebase development
```

**Comment by Prajwalism** [comment]
@spwoodcock sorry for the mistake, I will proceed to do as you mentioned.

**Comment by spwoodcock** [comment]
This looks good, but still includes commits from other PRs somehow.
E.g. https://github.com/hotosm/fmtm/pull/1300/commits/9cacda17e4f3ff6413c2c6ad13821fdb87aa37cc

Feel free to copy the code across to another PR if it helps 😄

**Review by spwoodcock — CHANGES_REQUESTED** [review]
see comment

**Comment by NSUWAL123** [comment]
@spwoodcock , I am closing this PR. I will make changes and push in another PR.

**Pull request closed without merging**